frontend/components/AddHostsModal/DownloadInstallers/DownloadInstallers.tsx

```diff
@@ -103,8 +103,9 @@ const DownloadForm: FunctionComponent<IDownloadFormProps> = ({
           className={`${baseClass}__button--download`}
           disabled={!selectedInstaller}
           type="submit"
+          isLoading={isCheckingForInstaller}
         >
-          {isCheckingForInstaller ? <Spinner /> : "Download installer"}
```

Comment:
Fixed bug of a legacy spinner that was incorrectly styled as it was left behind...